README.md

```diff
@@ -20,8 +20,7 @@ Health][res-ga4gh].
 ## About this repository
 
 This website is built using the [Next.js][res-next] framework for
-[React.js][res-react]. Most of the pages on the website are built statically
-during build time for fast client-side rendering and better SEO.
+[React.js][res-react]. Most of the pages on the website are built by Incremental Static Regeneration scheme on server for fast client-side rendering and better SEO.
```

Comment:
Can you please add a link for ISR and either remove "scheme" _or_ add a "the" in front of "Incremental"?

Also, maybe clearer to say "on the server" or "server-side", without "on".

After you can merge.